src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PrinterSettings.Windows.cs

```diff
@@ -614,7 +614,7 @@ public Graphics CreateMeasurementGraphics()
         }
 
         //whatever the call stack calling HardMarginX and HardMarginY here is safe
-        public Graphics CreateMeasurementGraphics(bool honorOriginAtMargins)
+        public Graphics? CreateMeasurementGraphics(bool honorOriginAtMargins)
         {
             Graphics g = CreateMeasurementGraphics();
             if (g != null && honorOriginAtMargins)
```

Comment:
`  Graphics g = CreateMeasurementGraphics();` wouldn't return null, i think this null check is redundant, we can remove it then the method will not return null

Reply:
Agreed, removing it.

Reply:
Same with line 641